scripts/ingests/filip_15.py

```diff
@@ -14,16 +14,17 @@
 # WRITE THE JSON FILES
 if SAVE_DB:
     db.save_database(directory='data/')
+#live google sheet
+link = "https://docs.google.com/spreadsheets/d/e/2PACX-1vTJZccUnKGOLcfPixaS3Tib0sqqC23n1Q_0ybjwXN4ZDYBT_-2_FeHLShVBzkIDfo39-tV1aoz5rCth/pub?output=csv"
 
-columns = ['Designation', 'Radius', 'log(g)', 'T_eff', 'Mass']  # columns wanted
-with open("scripts/ingests/table_9_filip_15.txt") as f:
-    content = f.readlines()
-    table_9 = Table.read(content, format='ascii', data_start=4, data_end=201, header_start=2, delimiter='\t',
-                         guess=False, fast_reader=False, include_names=columns)
+columns = ['Designation 9', 'pi', 'pi References', 'log(L_*/L_sun)', 'Age^a', 'Radius', 'log(g)', 'T_eff', 'Mass',
+           'R.A.', 'Decl.', 'Designation 1', 'Discovery References', 'SpT', 'SpT References']
+data9stuff = Table.read(link, format='ascii', data_start=6, data_end = 203, header_start=4, names=columns, guess=False,
+                        fast_reader=False, delimiter=',')
 
 data_columns = ['Radius', 'log(g)', 'T_eff', 'Mass']  # columns with data values
 
-for i in table_9:  # rewriting so that the empty values are None
+for i in data9stuff:  # rewriting so that the empty values are None
     for c in data_columns:
         if i[c] == 'cdots':
             i[c] = None
```

Comment:
```suggestion
for row in data9stuff:  # rewriting so that the empty values are None
    for column in data_columns:
        if row[column] == 'cdots':
            row[column] = None
```

Reply:
I forgot about this changing now